closure/goog/testing/net/xhrio_test.js

```diff
@@ -55,6 +55,44 @@ function testStaticSend() {
   assertEquals('sendInstances_ after READY', 0, sendInstances.length);
 }
 
+function testStaticSendWithFactory() {
+  var xhrFactoryCalled = 0, optionsFactoryCalled = 0, callbackInvoked = 0;
+  var wrapperFactory = new goog.net.WrapperXmlHttpFactory(
+      function() {
+        xhrFactoryCalled++;
+        return goog.net.XmlHttp();
+      },
+      function() {
+        optionsFactoryCalled++;
+        return {};
+      });
+
+  sendInstances = goog.testing.net.XhrIo.getSendInstances();
+  var returnedXhr = goog.testing.net.XhrIo.send(
+      'url', undefined, 'GET', '', undefined, 0, false, wrapperFactory);
+  assertEquals('sendInstances_ after send', 1, sendInstances.length);
+  xhr = sendInstances[sendInstances.length - 1];
+  assertTrue('isActive after request', xhr.isActive());
```

Comment:
Why does this need to be tested again if it was already tested in `testStaticSend`. It seems to me that most of the tests here are not affected by the custom factory.